### PR TITLE
feat: migration script to batch import existing orgs in CRM

### LINKED ISSUE #7355: validate existing org and user to import in CRM
### What
- We have a lot of orgs and users already in the pro platform and we want to create entries for them in the CRM.
- But we don't want to create entries and opportunities for all of them, we need a manual validation from the team.

For that we should dump a csv containing:
- orgs (name, contact info)
- users of orgs (name, emails, if possible, last login)
- date of creation
- number of products in the pro platform (and if possible, number of published products)
- last export date

So that the validation team can either:
- signal entries worth to create in the CRM
- find corresponding existing CRM entries and update off_org_id and off_user_id on them
We might as well keep it as a script / view (Admin only in this case) for future exports ;-) (stats and so on).

Part of:
- #7296

### LINKED ISSUE #7359: batch import existing org and users in CRM
### Blocked on
- #7355 

We can see, according to the number of entries, if we should make a script to create existing org, or if we do it manually (or something in between). 

### Part of
- #7296

### PR DESCRIPTION
**Pull request by 4nt0ineB**
### What

Existing orgs have been manually reviewed by @manon-corneille, using a dump generated by this script https://github.com/openfoodfacts/openfoodfacts-server/pull/10507.
This PR provides a migration script (`scripts/migration/2024_08_validate_and_sync_crm_existing_orgs.pl`)  to accept the selected orgs. The others are rejected and receive a rejection email notification.

- added sync of org's country
- few fixes in CRM.pm 

- [x]  Awaiting email's wording.

### Related issue(s) and discussion

- Fixes #7355
- Fixes #7359

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Looks good to me

File: scripts/migrations/2024_08_validate_and_sync_crm_existing_orgs.pl [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+		$org_ref->{valid_org} = 'rejected';
+	}
+
+	store_org($org_ref);
+	if ($org_is_valid) {
+		print "$org_id\n";
+	}
+}
+
+sub accepted_orgs {
```
Comment (alexgarel) [inline scripts/migrations/2024_08_validate_and_sync_crm_existing_orgs.pl:69]: could have been in a separate CSV @4nt0ineB !

**Comment by alexgarel** [comment]
Kudos @4nt0ineB ! With that you really are about to achieve 100% of your internship objectives !

File: scripts/migrations/2024_08_validate_and_sync_crm_existing_orgs.pl (hunk 1 above)
Comment (4nt0ineB) [reply scripts/migrations/2024_08_validate_and_sync_crm_existing_orgs.pl:69]: yeah, ok might have been lazy on this x). I should probably use a csv and use it as a check point

File: scripts/migrations/input/2024_08_orgs_to_accept_and_sync
Comment (alexgarel) [inline scripts/migrations/input/2024_08_orgs_to_accept_and_sync:1]: I would not put this in git.

**Review by stephanegigandet — APPROVED** [review]
Looks good!

**Pull request merged**